flask_multipass/providers/ldap/util.py

```diff
@@ -8,7 +8,6 @@
 
 from collections import namedtuple
 from contextlib import contextmanager
-from urlparse import urlparse
```

Comment:
Since flask-multipass package depends on Flask, it implicitly depends on Werkzeug, so you could just use `from werkzeug.urls import url_parse`

https://werkzeug.palletsprojects.com/en/0.16.x/urls/#werkzeug.urls.url_parse